vividus-plugin-applitools/src/main/resources/applitools-visual-comparison.ftl

```diff
@@ -4,7 +4,28 @@
   background-color: #1DA69B;
   box-shadow: 0px 8px 15px rgba(0, 0, 0, 0.1);
 }
+
+.dot {
+  height: 13px;
+  width: 13px;
+  border-radius: 50%;
+  display: inline-block;
+}
 </#macro>
+
+<#macro custom_section>
+ <#if result.applitoolsTestResults.accessibilityCheckResult??>
+     <#assign checkResult = result.applitoolsTestResults.accessibilityCheckResult>
+     <#assign statusColor = checkResult.isPassed()?then('green', 'red')>
```

Comment:
I believe it was agreement to use common styles (including colors) for all attachments

Reply:
discussed